batchflow/models/tf/nn/train.py

```diff
@@ -0,0 +1,97 @@
+""" Helpers for training """
+from math import pi
+
+import tensorflow as tf
+from tensorflow.math import sin, asin, floor # pylint: disable=import-error
```

Comment:
Each of these functions if used pretty much only once, so it would be better to call them as `tf.math.<>`

Reply:
Why?

Reply:
@SergeyTsimfer, Done

Reply:
> Why?

The next person to add smth to this file will need to either add more function (and thus clutter the beginning of the file), or change usage to directly to `tf`, as I suggested